lab_1/main.py

```diff
@@ -69,7 +107,26 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    concordance = []
+    bad_inputs = [[], {}, 'string', (), None, 9.34, True, [None]]
+    for index, token in enumerate(tokens):
+        if token not in bad_inputs:
+            if token == word:
+                if (left_context_size > 1) & (right_context_size > 1):
+                    context = tokens[index - left_context_size:index] + tokens[index: index + right_context_size + 1]
+                    concordance.append(context)
+                elif (left_context_size > 1) & (right_context_size < 1):
```

Comment:
Do you need the right statement? if right_context_size is > 1, the first "if" will work.